source/shared/core_conn.cpp

```diff
@@ -150,21 +150,55 @@ sqlsrv_conn* core_sqlsrv_connect( _In_ sqlsrv_context& henv_cp, _In_ sqlsrv_cont
     
     build_connection_string_and_set_conn_attr( conn, server, uid, pwd, options_ht, valid_conn_opts, driver, conn_str TSRMLS_CC );
     
-    bool is_missing_driver = false;
+    bool is_missing_driver = true;
 
     if ( conn->is_driver_set ) {
         r = core_odbc_connect( conn, conn_str, is_missing_driver, is_pooled );
```

Comment:
What happens when user set driver to ODBC 13 and ce_option.enabled is true? Will there be an error? Maybe should check for if ( conn-> is_driver_set && conn->ce_option.enabled ) first.